Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/donkpocket.yml

```diff
@@ -58,7 +58,9 @@
         maxVol: 10
         reagents:
         - ReagentId: Nutriment
-          Quantity: 7
+          Quantity: 8
```

Comment:
Can you please explain why you change that?

Reply:
This is not explained in the PR, but the comment on line 27 says warm should be +5 nutriment compared to the base. The base for this donk pocket is 3, given on line 17.